File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TextBoxes_View.cpp

#include "TextBoxes_View.h"
#include "TextBoxes_View.h"
#include <vector>
#include <string>

namespace widgetassertions
{
  std::string TextBoxes_View::getMyValueTextBoxText( ) 
  {
    return this->MyValueTextBoxText;
  }
  bool TextBoxes_View::getIsMyValueTextBoxVisible( ) 
  {
    return this->isMyValueTextBoxVisible;
  }
  bool TextBoxes_View::getIsMyValueTextBoxEnabled( ) 
  {
    return this->isMyValueTextBoxEnabled;
  }
  std::vector<std::string> TextBoxes_View::getMultiLineTextBoxValueTextBoxText( ) 
  {
    return this->MultiLineTextBoxValueTextBoxText;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TextBoxes_View.cs
/// <filename>
///     TextBoxes_View.cs
/// </filename>
using System.Collections.Generic;

namespace widgetassertions
{
    public class TextBoxes_View
  {
        private string MyValueTextBoxText;
     
        public     virtual string getMyValueTextBoxText()
    {
      return this.MyValueTextBoxText;
    }
     
        private bool isMyValueTextBoxVisible;
     
        public     virtual bool getIsMyValueTextBoxVisible()
    {
      return this.isMyValueTextBoxVisible;
    }
     
        private bool isMyValueTextBoxEnabled;
     
        public     virtual bool getIsMyValueTextBoxEnabled()
    {
      return this.isMyValueTextBoxEnabled;
    }
     
        private System.Collections.Generic.List<string> MultiLineTextBoxValueTextBoxText = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getMultiLineTextBoxValueTextBoxText()
    {
      return this.MultiLineTextBoxValueTextBoxText;
    }
  }
}

